.ci/install_clearcontainers_kernel.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+repo_owner="clearcontainers"
+repo_name="linux"
+cc_linux_releases_url="https://github.com/${repo_owner}/${repo_name}/releases"
+
+set -e
+function usage() {
+    cat << EOT
+Usage: $0 <version>
+Install the containers clear kernel image <version> from clearcontainers/linux.
+
+version: Use latest to pull latest kernel or a version from ${cc_linux_releases_url}
```

Comment:
Nit: I'd make that `'latest'` to make it clearer that the word is an argument.

Also, please can you add some words explaining:

- what process is uploading these binaries to github.
- how the kernel version relates to the `clear_container_kernel=` variable in https://github.com/clearcontainers/runtime/blob/master/versions.txt.

Reply:
yep, looks better -  I added a new commit to document what you ask here. Please take look and let me know if is clear and the answers to this questions.